frontend/src/bundles/auth/components/AdditionalForm.vue

```diff
@@ -1,15 +1,51 @@
 <template>
-  <form :value="!invalid" @submit.prevent="handleSubmit">
+  <form
+    :value="!invalid"
+    @submit.prevent="handleSubmit"
+  >
+    <validation-provider
+      name="gender"
+    >
+      <p>You are <span class="primary--text font-weight-bold">{{ gender || 'male' }}</span></p>
```

Comment:
Avoid using logic in the template
Also you have a default value for this statement

Reply:
fixed